### ISSUE TITLE
[BETA]: (beta-2) TypeError in PdkDeliveryOptionsMigration

### ISSUE THREAD

**Issue by mariuszsienkiewicz** [myparcelnl/prestashop#244]
### Version check

- [X] My version of the plugin is v4.0.0-beta.0 or above.

### PrestaShop version

8.1.5

### PHP version

8.1

### What went wrong?

https://github.com/myparcelnl/prestashop/blob/5d0da006167dd2a6840073118679eecc00edbc7f/src/Migration/Pdk/PdkDeliveryOptionsMigration.php#L125-L128

```php
// lets say that $row['delivery_settings'] is null, then json_decode('', true) => NULL
$deliverySettings = json_decode($row['delivery_settings'] ?? '', true); 

// array_merge(NULL, []) => Uncaught TypeError: array_merge(): Argument #1 must be of type array, null given
$newDeliveryOptions = $this->transformDeliveryOptions(array_merge($deliverySettings, $extraOptions)); 

// same with $extraOptions
```

### Reproduction steps

none

**Comment by mariuszsienkiewicz** [comment]
Okay I see that you are already working on it in #241.

**Comment by EdieLemoine** [comment]
Resolved in #241 (expect a new release very soon, hopefully this week 🤞🏼)
